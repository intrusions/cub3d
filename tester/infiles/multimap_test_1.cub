NO         				 ./path_to_the_north_texture









C																			225,30,0










SO 																				./path_to_the_south_texture
SO 																				./path_to_the_south_texture
WE ./path_to_the_west_texture

EA 										            ./path_to_the_east_texture


F							 220,100,0









0000000000000000000000000
1000000000110000000000001
1011000001110000000000001
1001000011111111000000000000S0001
111111111N   N111
1111111111111111
1111111110110000011100001
100000000011000001110111111111111
11111111111111011100000010001
11110111111111111101010010001
1100000011010101110000001001
1000000000000N0011000000100
10000000000000001101010010N01
1100000111010101111101111000111
11110111 1110101 101111010001
111111111111111      11111111111111
