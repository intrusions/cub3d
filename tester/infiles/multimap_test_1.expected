Loading scene...
Loading map...
Starting game...
